cmd/neofs-oauthz/service.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+const defaultShutdownTimeout = 15 * time.Second
+
+type (
+	// Service serves metrics.
+	Service struct {
+		*http.Server
+		enabled bool
+		log     *zap.Logger
+	}
+
+	// Services is a collection for services which can be started in background.
+	Services struct {
+		services []*Service
+	}
+)
+
+// NewService is a constructor for Service.
+func NewService(
```

Comment:
Doesn't need to be exported (and everything else here as well).

Reply:
Updated